src/renderer/viz/expressions/RampImage.js

```diff
@@ -0,0 +1,88 @@
+import defaultSVGs from '../defaultSVGs';
+import { checkType, checkInstance } from './utils';
+import Property from './basic/property';
+import CategoryIndex from './CategoryIndex';
+import ListImage from './ListImage';
+import SVG from './SVG';
+import Base from './base';
+import { OTHERS_GLSL_VALUE, DEFAULT_OPTIONS, DEFAULT_OTHERS } from './constants';
+
+export default class RampImage extends Base {
+    _bindMetadata (metadata) {
+        Base.prototype._bindMetadata.call(this, metadata);
+        this.type = this.palette.childType;
+
+        if (this.others === DEFAULT_OTHERS) {
+            this.others = new SVG(defaultSVGs.circle);
```

Comment:
a constant for this?

Reply:
we should add all the default values in constants as you mentioned before :)